### ISSUE TITLE
Add staticcheck to golang type

### ISSUE THREAD

**Issue by arthurnn** [AppThreat/sast-scan#17]
I see the Dockerfile is installing `staticcheck` tool, however I dont see it running on the golang type: https://github.com/AppThreat/sast-scan/blob/4ea2ce417db47fb330a5e99bc5548dad01d6497d/lib/config.py#L122-L129

Is it because the SARIF converter is not yet ready for staticcheck?

Thanks

**Comment by prabhu** [comment]
Correct. Will add this issue to my list for the week.

**Comment by prabhu** [comment]
Still not done with SARIF for staticcheck. Actually, got distracted and trying to build a test tool for testing sast-scan since I am currently check all project types and CI integrations manually :(

But here is a secret. If you create a local .sastscanrc you can override pretty much anything including replacing or including staticcheck in addition to gosec for golang.

https://github.com/AppThreat/sast-scan/blob/master/docs/TRICKS.md#config-file

For the key golang, if the value is a list then it is considered as a single tool. If instead the value is a map (for eg: [depscan](https://github.com/AppThreat/sast-scan/blob/master/lib/config.py#L110)) then the tool would execute one tool at a time.

So the config for golang would resemble

```json
"golang": {
  "gosec": [go sec args],
  "staticcheck": [static check args]
}
```

Let me know if this workaround helps for now.

**Comment by prabhu** [comment]
staticcheck is now available with version 1.1.1

```bash
tool         description                        critical    high    medium    low  status      total
-----------  -------------------------------  ----------  ------  --------  -----  --------  -------
staticcheck  Go static analysis                        7       0         0      0  ❌               7
gosec        Golang security checks by gosec           0       0         1      0  ✅               1
```
